scripts/generate_usb_vif/constants/dt_constants.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""This file contains device tree constants defined to be used by generate_vif.py"""
+
+SINK_PDOS = "sink-pdos"
+PD_DISABLE = "pd-disable"
+POWER_ROLE = "power-role"
+
+DT_VIF_ELEMENTS = {
+    SINK_PDOS: "SnkPdoList",
+    PD_DISABLE: "USB_PD_Support",
```

Comment:
I find it odd that `USB_PD_Support` has a constant name of `PD_DISABLE`, and it takes its value as the _opposite_ of the `PD_DISABLE` DT property. I would suggest naming the VIF element constant after the VIF elements. Seeing `PD_SUPPORT = not PD_DISABLE` in the code would make more sense intuitively.

Also, it would make more sense to me if these VIF element names went in `vif_element_constants.py. You could import that file from here to use those constant names in `DT_VIF_ELEMENTS`.

(I also find it odd that the DT property is `pd-disable` and not something like `pd-enable` or `pd-support` that would be true when PD is enabled, but that's outside this scope of this PR.)

Reply:
The above dict contains mapping from DT prop names to VIF element name, which code will translate and create XML element. I avoided creating constants in vif_elemnts_constants.py file for them("SnkPdoList", "USB_PD_Support"), as it's used only once and not used by script. 

I will remove the mapping of PD_DISABLE one

Reply:
Removed PD_DISABLE mapping in new revision